src/constants/max-file-size.constant.ts

```diff
@@ -0,0 +1 @@
+export const MAX_FILE_SIZE = 12000000;
```

Comment:
Can we rename it to `MAX_FILE_SIZE_BYTES` and set the value to `12582912` so we are using `1024` as a base instead of `1000` for improved accuracy.

Reply:
Changed